libraries/SocketWrapper/SocketWrapper.h

```diff
@@ -105,21 +105,24 @@ class ZephyrSocketWrapper {
 	bool connectSSL(const char *host, uint16_t port, char *ca_certificate_pem = nullptr) {
```

Comment:
This should be `const char *ca_certificate_pem` together with 

`int connectSSL(const char *host, uint16_t port, const char *cert)` in ZephyrClient.h and
`int connect(const char *host, uint16_t port, const char *cert)` in ZephyrSSLClient.h

or we need to cast the certificate blob